packages/api/models/rules.js

```diff
@@ -556,11 +556,11 @@ class Rule extends Manager {
     }
 
     // update rule meta
-    if (!get(rule, 'meta.visibilityTimeout')) {
+    if (get(rule, 'meta.visibilityTimeout') === undefined) {
```

Comment:
👍🏻  We should add a comment to the phase 3 feature ticket (or PR if it's in flight) to make sure this gets fixed wherever it is moved/this change isn't reverted for the phase 3 merge.

Reply:
Although I suppose the new units will fail.  🤔